views/improvementProjectTemplate.ejs

```diff
@@ -438,8 +438,7 @@
                   }
                   
                   %>
-                  <% if( ((i+1)%2==0) && skipHeader==true
-                    && data.tasksArray[i-1].attachments && data.tasksArray[i-1].attachments.length ==0 && data.tasksArray[i-1].children && data.tasksArray[i-1].children.length == 0){ } else { %>
+                  <% if( ((i+1)%2==0) && skipHeader==true && data.tasksArray[i-1].children && data.tasksArray[i-1].children.length == 0){ } else { %>
```

Comment:
@MallanagoudaB what happens if task array has attachments?

Reply:
gagan said that should be shown if there are no attachedment and subtask

Reply:
@MallanagoudaB that is not my question is if task array childeren is null but attachment is present will this change break the pdf? did you test that?